styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container hidden-xs hidden-sm">
   <div class="row">
     <div class="col-sm-3">
-      <div class="col-sm-3">
-        <h5>We've got your back</h5>
-        <h4 class="text34">24x7x365</h4>
+      <h5 class="rug-subhead-medium no-margin">We've got your back</h5>
+      <h4 class="rug-subhead-large no-margin">24<small class="white">x</small>7<small class="white">x</small>365</h4>
```

Comment:
@bighappyface Is there colors set on small elements within h4s? That seems very weirdly specific. The rule I used should just make everything within the rug white? Can we chat about this today at some point? I may not be understanding why were setting the color this way, it seems clunky.